### ISSUE TITLE
What happened to the addBusinessDays function?

### ISSUE THREAD

**Issue by davidhenley** [date-fns/date-fns#3741]
At 3.0.0 this was removed from the docs? What am I missing?

**Comment by fturmel** [comment]
Good catch! It looks like a bad `@category` jsdoc value at first glance, which would exclude the function from the docs.

Maybe we need some tests around this.

https://github.com/date-fns/date-fns/blob/8b637617a77d7284d2c4c3ba56ef463f0fcaa199/src/addBusinessDays/index.ts#L9

https://github.com/date-fns/date-fns/blob/8b637617a77d7284d2c4c3ba56ef463f0fcaa199/src/subBusinessDays/index.ts#L5

https://github.com/date-fns/date-fns/blob/8b637617a77d7284d2c4c3ba56ef463f0fcaa199/docs/config.js#L7-L28

**Comment by davidhenley** [comment]
I created a pull request

**Comment by kossnocorp** [comment]
Thank you! Closed via https://github.com/date-fns/date-fns/commit/5c1adb5369805ff552737bf8017dbe07f559b0c6. It will go live with the next release.
